lib/matplotlib/scale.py

```diff
@@ -1,16 +1,36 @@
 """
 Scales define the distribution of data values on an axis, e.g. a log scaling.
-They are defined as subclasses of `ScaleBase`.
 
-See also `.axes.Axes.set_xscale` and the scales examples in the documentation.
+Scales are subclasses `ScaleBase`. Their mapping is implemented through `.Transform`
+subclasses that are separable and have an inverse.
 
-See :doc:`/gallery/scales/custom_scale` for a full example of defining a custom
-scale.
+The following scales are builtin:
+
+============= ===================== ================================ =================================
+Name          Class                 Transform                        Inverted transform
+============= ===================== ================================ =================================
+"asinh"       `AsinhScale`          `AsinhTransform`                 `InvertedAsinhTransform`
+"function"    `FuncScale`           `FuncTransform`                  `FuncTransform`
+"functionlog" `FuncScaleLog`        `FuncTransform` + `LogTransform` `InvertedLogTransform` + `FuncTransform`
+"linear"      `LinearScale`         `.IdentityTransform`             `.IdentityTransform`
+"log"         `LogScale`            `LogTransform`                   `InvertedLogTransform`
+"logit"       `LogitScale`          `LogitTransform`                 `LogisticTransform`
+"symlog"      `SymmetricalLogScale` `SymmetricalLogTransform`        `InvertedSymmetricalLogTransform`
+============= ===================== ================================ =================================
+
+A user will often only use the scale name, e.g. when setting the scale through
+`~.Axes.set_xscale`: ``ax.set_xscale("log")``.
+
+See also the :ref:`scales examples <sphx_glr_gallery_scales>` in the documentation.
+
+Custom scaling can be achieved through `FuncScale`, or by creating your own
+`ScaleBase` subclass (see :doc:`/gallery/scales/custom_scale`).
+Third parties can register their scales by name through `register_scale`.
 
 Matplotlib also supports non-separable transformations that operate on both
```

Comment:
I find this sentence pretty confusing on its own.  Appreciate you are just changing the formatting here, but if we are improving things, we might mention that the previous scales act on one axis at a time, and are for Cartesian axes.

Reply:
> , and are for Cartesian axes.

This is not quite true, you can e.g set a log radial axis in a polar plot.

I'm tempted to leave out the paragraph. I think that scales and projections are mostly independent of each other (but am not completely sure).

Reply:
Ok fair enough.  Maybe "scales are typically applied individually to the axis of a separable axes. Non-separable axes are dealt with using projections".  But maybe you are correct that you could do without it here.

Reply:
I've removed that paragraph. Irrespective of the exact technical relationship. People looking at scales will typically have a certain projection (mostly cartesian axes) in mind and only consider the scaling behavior along the given axes. It's likely not helpful for them to bring up the topic of other projections here.